log4j-slf4j-impl/src/main/java/org/apache/logging/slf4j/package-info.java

```diff
@@ -21,7 +21,7 @@
  */
 @Export
 @Header(name = Constants.BUNDLE_ACTIVATIONPOLICY, value = Constants.ACTIVATION_LAZY)
-@Version("2.21.0")
+@Version("2.23.0")
```

Comment:
I guess this should have a different value on `3.x` branch?

Reply:
@PascalSchumacher,

thanks for the remark.

We don't bump package versions automatically (if there are no minor/major changes we leave them the same, but in this case BND remarks that we have hidden (removed `public`) from two classes:

```
[ERROR] * org.apache.logging.slf4j                           PACKAGE    MAJOR      2.23.0     2.21.0     3.0.0      -
[ERROR]   MAJOR                PACKAGE    org.apache.logging.slf4j
...
[ERROR]     MAJOR              CLASS      org.apache.logging.slf4j.Log4jLogger
[ERROR]       REMOVED          IMPLEMENTS java.io.Serializable
[ERROR]     REMOVED            CLASS      org.apache.logging.slf4j.Log4jMarker
```